lib/wolfe/cleanup.rb

```diff
@@ -9,6 +9,7 @@ class Cleanup
     def initialize(configuration)
       @configuration = configuration
       validate_configuration
+      @first_relevant_date = Date.today - 5.years
```

Comment:
Is there a specific reason for why this time span was chosen? It should also be extracted into a meaningful constant, like `BACKUP_FIRST_CUTOFF_DATE`, especially if it should be a fixed date.